### PR TITLE
quick fix for type mistmatch

### PR DESCRIPTION
**Pull request by DavideTisi**
fix for #247 

----
:books: Documentation preview :books:: https://equistore--250.org.readthedocs.build/en/250/

### CODE REVIEW

**Comment by DavideTisi** [comment]
it should fix the problem but it is not tested yet

File: python/src/equistore/operations/_dispatch.py [hunk 1]
```diff
@@ -65,6 +65,7 @@ def bincount(input, weights=None, minlength=0):
                                         Defaults to None.
         minlength (int, optional): A minimum number of bins for the output array.
                                         Defaults to 0.
+        out_format
```
Comment (Luthaf) [inline python/src/equistore/operations/_dispatch.py:68]: looks like a leftover

Comment (Luthaf) [reply python/src/equistore/operations/_dispatch.py:68]: Could you also add a link to `array_like_data` here? Pointing future devs to this problem =)

**Review by Luthaf — COMMENTED** [review]
You've checked elsewhere that there are no other usage of bincount, right?

**Comment by Luthaf** [comment]
@abmazitov I'll let you check this fixes the issue!

File: python/src/equistore/operations/_dispatch.py (hunk 1 above)
Comment (DavideTisi) [reply python/src/equistore/operations/_dispatch.py]: not sure what you meant but i added a comment

**Comment by DavideTisi** [comment]
i rebased and fixed the linter with `blackdoc`. For me it can be merged

File: python/src/equistore/operations/_dispatch.py (hunk 1 above)
Comment (DavideTisi) [reply python/src/equistore/operations/_dispatch.py]: 👍

**Pull request merged**